src/mono/CMakeLists.txt

```diff
@@ -862,6 +862,7 @@ endif()
 ### End of OS specific checks
 
 include_directories("${CLR_SRC_NATIVE_DIR}")
+include_directories("${CLR_SRC_NATIVE_DIR}/public")
```

Comment:
@lambdageek Not sure how this worked previously.

Reply:
It is done by `add_subdirectory("${CLR_SRC_NATIVE_DIR}/public" public_apis)` a few lines below.